### DISCUSSION TITLE
Branch chamada-2021 não encontrada

### DISCUSSION THREAD

**Discussion by felipeafreitas** [Nexo-Tec/time#12] [Q&A] [answered]
Boa noite! Tudo jóia? Gostaria de fazer 2 perguntas:

- Fui realizar o pull-request para entrega da aplicação, porém, não encontrei a branch "chamada-2021". Como posso seguir?
- Por onde posso enviar os links do projeto realizado? Posso incluí-los diretamente no arquivo markdown da minha aplicação?

Obrigado desde já!
abs,
F.

**Comment by ibrahimcesar** [comment, marked as answer]
Olá @sincopeiro ,

Nós não criamos uma branch com esse nome, você pode simplesmente fazer o fork do nosso repositório acrescentar seu arquivo em sua versão e então abrir um _pull request_ da sua versão para a nossa branch _main_.

**Reply by felipeafreitas** [reply]
Show! Muito obrigado =]
